### PR TITLE
Allow creation of users via REST API 

### PR DESCRIPTION
**Pull request by betsybeth**
##### Login to the application
image
##### click on the navbar
image
##### scroll and click on the REST API
image
##### scroll down the REST API file click on browser API
image
##### Click on the underlined url
image
##### You will find creating users via REST API 
image
##### You can create users via REST API
image
##### If you are not  a user in the wger application the REST API should respond with:
image

### CODE REVIEW

**Review by meshnesh — DISMISSED** [review]
Nice implementation of your task and completion.
Please Resolve the conflicts raised in the PR, for the approval

**Comment by betsybeth** [comment]
Merge conflict solved

File: wger/core/api/views.py [hunk 1]
```diff
@@ -70,6 +71,55 @@ def username(self, request, pk):
         return Response(UsernameSerializer(user).data)
 
 
+class UserApiRegistrationViewset(viewsets.ModelViewSet):
```
Comment (FelixWambiri) [inline wger/core/api/views.py:74]: Add comments to describe what is going on in this file

**Review by FelixWambiri — COMMENTED** [review]
Please add more elaborate comments.

File: wger/core/api/views.py (hunk 1 above)
Comment (betsybeth) [reply wger/core/api/views.py:74]: I have commented the file kindly check out my files

**Review by FelixWambiri — COMMENTED** [review]
Your comments should tell us what the code does and for complex code blocks how.

**Comment by betsybeth** [comment]
Solved.Thank you

**Review by FelixWambiri — APPROVED** [review]
Nice work

**Pull request merged**